src/realm/object-store/property.hpp

```diff
@@ -107,9 +110,13 @@ struct Property {
     // Text property with fulltext index
     Property(std::string name, IsFulltextIndexed indexed, std::string public_name = "");
 
+    // Link to another object
```

Comment:
How to have a nested collection of links to another object?

Reply:
Added a new ctor.